src/main/resources/public/js/main.js

```diff
@@ -1 +1,122 @@
-console.log("l");
\ No newline at end of file
+const MAXIMUM_OF_ANSWERS_TO_SHOW = 5;
+$(window).on("load", function() {
+
+    var actualQuestion = 0;
+    var numberOfQuestions;
+    var roundScore = 0;
+    var team1Score = 0, team2Score = 0;
+    var round;
+
+    async function main() {
+        round = await getRoundInformation();
+        numberOfQuestions = round.questionsNumber;
+        let firstQuestion = round.questions[actualQuestion];
+        displayQuestion(firstQuestion);
+        addListener2AnswerButtons();
+        go2NextQuestion();
+        assignScore2Team();
+        showWrongAnswerMessage();
+        setInitialScore2Boards();
```

Comment:
El uso de "2" para representar la palabra "to" lo usamos cuando convertimos algo de un formato a otro como puedes verlo en nuestra [guía de estilo](https://islasgeci.github.io/ciencia_de_datos/guia_de_estilo/nomenclatura.html). 
Por ejemplo, podrías cambiar `setInitialScore2Boards` por `setInitialScoreOnBoards`.

Reply:
Sí gracias, también hice el cambio de los nombres de las demás funciones que aún tenían "2"